projects/dotcms-field-elements/src/models/dot-CMS-temp-file.ts

```diff
@@ -0,0 +1,10 @@
+export interface DotCMSTempFile {
```

Comment:
You sure about this file name?

Reply:
actually no, but just followed the name in the issue.